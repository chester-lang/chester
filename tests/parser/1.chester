
cons2 = Cons %{ head = 0, tail = Nil };
