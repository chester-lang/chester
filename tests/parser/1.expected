Block(
  statements = Vector(
    OpSeq(
      seq = Vector(
        Identifier(name = "cons2", meta = None),
        Identifier(name = "=", meta = None),
        Identifier(name = "Cons", meta = None),
        ObjectExpr(
          clauses = Vector(
            ObjectExprClause(
              key = Identifier(name = "head", meta = None),
              value = IntegerLiteral(value = 0, meta = None),
              meta = None
            ),
            ObjectExprClause(
              key = Identifier(name = "tail", meta = None),
              value = Identifier(name = "Nil", meta = None),
              meta = None
            )
          ),
          meta = None
        )
      ),
      meta = None
    )
  ),
  result = None,
  meta = None
)